<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>589</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>TrimStart</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Files to Process</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="listWidget_files">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DropOnly</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QToolButton" name="btn_add_file">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Media Files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/ic/res/icon/ic_add_black_48dp.png</normaloff>:/ic/res/icon/ic_add_black_48dp.png</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="btn_remove_file">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove Selected Entries&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/ic/res/icon/ic_remove_black_48dp.png</normaloff>:/ic/res/icon/ic_remove_black_48dp.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QToolButton" name="btn_selectAll">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select All&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/ic/res/icon/ic_border_all_black_48dp.png</normaloff>:/ic/res/icon/ic_border_all_black_48dp.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QToolButton" name="btn_selectNone">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear Selection&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/ic/res/icon/ic_border_clear_black_48dp.png</normaloff>:/ic/res/icon/ic_border_clear_black_48dp.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Trim </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_secends_to_trim">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>seconds from the beginning.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Save to</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radio_original_folder">
             <property name="text">
              <string>Source Folder</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_other_folder">
             <property name="text">
              <string>Other Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineedit_folder_path">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_choose_path">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>..</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>If left blank, files will be stored in source folder.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>File Surfix</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_file_surfix">
           <property name="text">
            <string>-trim</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>File surfix must be specified if you choose to save in source folder.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAdd_Files"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionLocate_ffmpeg_exe"/>
    <addaction name="separator"/>
    <addaction name="actionGet_ffmpeg_exe"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout_FFmpeg"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAdd_Files">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_add_black_48dp.png</normaloff>:/ic/res/icon/ic_add_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Files...</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_remove_black_48dp.png</normaloff>:/ic/res/icon/ic_remove_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Selected Entries...</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_exit_to_app_black_48dp.png</normaloff>:/ic/res/icon/ic_exit_to_app_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionLocate_ffmpeg_exe">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_extension_black_48dp.png</normaloff>:/ic/res/icon/ic_extension_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Locate ffmpeg.exe</string>
   </property>
  </action>
  <action name="actionGet_ffmpeg_exe">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_public_black_48dp.png</normaloff>:/ic/res/icon/ic_public_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Get FFmpeg</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_error_outline_black_48dp.png</normaloff>:/ic/res/icon/ic_error_outline_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Qt ...</string>
   </property>
   <property name="toolTip">
    <string> About Qt</string>
   </property>
  </action>
  <action name="actionAbout_FFmpeg">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_link_black_48dp.png</normaloff>:/ic/res/icon/ic_link_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;FFmpeg</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ic/res/icon/ic_info_outline_black_48dp.png</normaloff>:/ic/res/icon/ic_info_outline_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radio_other_folder</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_choose_path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
